package.json

```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "react": "^16.8.0",
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
```

Comment:
Can we shorten it to`"react": ">=16.8.0"`?

Reply:
I think we're removing the peer dependencies all together